src/deb/nginx/nginx.conf

```diff
@@ -46,48 +47,49 @@ http {
 	proxy_set_header              X-Real-IP $remote_addr;
 	proxy_set_header              X-Forwarded-For $proxy_add_x_forwarded_for;
 	proxy_pass_header             Set-Cookie;
-	proxy_buffers                 32 4k;
-	proxy_buffer_size             8k;
+	proxy_buffers                 256 4k;
+	proxy_buffer_size             32k;
+	proxy_busy_buffers_size       32k;
+	proxy_temp_file_write_size    256k;
 	proxy_connect_timeout         30s;
 	proxy_read_timeout            300s;
 	proxy_send_timeout            180s;
 	# Log format
-	log_format                    main '$remote_addr - $remote_user [$time_local] $request '
-	  '"$status" $body_bytes_sent "$http_referer" ' '"$http_user_agent" "$http_x_forwarded_for"';
-	log_format                    bytes '$body_bytes_sent';
+	log_format                    main '$remote_addr - $remote_user [$time_local] $request "$status" $body_bytes_sent "$http_referer" "$http_user_agent" "$http_x_forwarded_for"';
 	access_log                    /var/log/hestia/nginx-access.log main;
 	# Mime settings
-	include                       /usr/local/hestia/nginx/conf/mime.types;
+	include                       mime.types;
 	default_type                  application/octet-stream;
 	# Compression
 	gzip                          on;
 	gzip_vary                     on;
 	gzip_comp_level               6;
 	gzip_min_length               1024;
-	gzip_buffers                  16 8k;
+	gzip_buffers                  128 4k;
 	gzip_http_version             1.1;
-	gzip_types                    text/plain text/css text/javascript text/js text/xml
-	  application/json application/javascript application/x-javascript application/xml
-	  application/xml+rss application/x-font-ttf image/svg+xml font/opentype;
+	gzip_types                    text/css text/javascript text/js text/plain text/richtext text/shtml text/x-component text/x-java-source text/x-markdown text/x-script text/xml image/bmp image/svg+xml image/vnd.microsoft.icon image/x-icon font/otf font/ttf font/x-woff multipart/bag multipart/mixed application/eot application/font application/font-sfnt application/font-woff application/javascript application/javascript-binast application/json application/ld+json application/manifest+json application/opentype application/otf application/rss+xml application/ttf application/truetype application/vnd.api+json application/vnd.ms-fontobject application/wasm application/xhtml+xml application/xml application/xml+rss application/x-httpd-cgi application/x-javascript application/x-opentype application/x-otf application/x-perl application/x-protobuf application/x-ttf;
 	gzip_proxied                  any;
-	gzip_disable                  "MSIE [1-6]\.";
+	gzip_disable                  "msie6";
```

Comment:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=867024

Maybe it is time to just remove it... 

Microsoft IE 6 has been dead for way to many years

Reply:
Agree to remove it by default.

I'm going to bed, I'll finish other parts later.